notification/method/webpush.php

```diff
@@ -97,8 +98,9 @@ public function get_type(): string
 	*/
 	public function is_available(type_interface $notification_type = null): bool
 	{
-		return parent::is_available($notification_type) && $this->config['wpn_webpush_enable']
-			&& !empty($this->config['wpn_webpush_vapid_public']) && !empty($this->config['wpn_webpush_vapid_private']);
+		return $this->config['wpn_webpush_enable']
```

Comment:
maybe we use `config->offsetGet()` here, just because it has an isset check which should be helpful just in case...because you know...users.

Reply:
Yes, makes sense.